<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>16</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="startWavlengthLabel">
        <property name="text">
         <string>Start Wavlength (nm)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="startWavlengthDoubleSpinBox">
        <property name="minimum">
         <double>600.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1750.000000000000000</double>
        </property>
        <property name="value">
         <double>1500.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="stopWavelengthLabel">
        <property name="text">
         <string>Stop Wavelength (nm)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="stopWavelengthDoubleSpinBox">
        <property name="minimum">
         <double>600.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1750.000000000000000</double>
        </property>
        <property name="value">
         <double>1600.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sensitivityLabel">
        <property name="text">
         <string>Sensitivity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sensitivityComboBox">
        <item>
         <property name="text">
          <string>Hold</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Auto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>High 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>High 2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>High 3</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="referenceLevelLabel">
        <property name="text">
         <string>Reference Level (dBm)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="referenceLevelDoubleSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-90.000000000000000</double>
        </property>
        <property name="maximum">
         <double>20.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="resoltuionNmLabel">
        <property name="text">
         <string>Resoltuion (nm)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="resoltuionNmDoubleSpinBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="SweepPushButton">
        <property name="text">
         <string>Sweep</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="RepeatSweepPushButton">
        <property name="text">
         <string>Continuous sweep</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="StopRepeatPushButton">
        <property name="text">
         <string>Stop continuous</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,1">
      <item>
       <widget class="PlotWidget" name="plotWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Visible</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListView" name="listView">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>Calculate total power per trace</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Trace as reference</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="DeletePushButton">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="SavePushButton">
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1007</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>818</x>
     <y>644</y>
    </hint>
    <hint type="destinationlabel">
     <x>822</x>
     <y>662</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
